libraries/engage/locations/components/FulfillmentSelector/FulfillmentSelector.jsx

```diff
@@ -25,10 +29,11 @@ import * as styles from './FulfillmentSelector.style';
  * @param {Object} props.location Last location that was selected for the previous product/variant.
  * @returns {JSX}
  */
-export const FulfillmentSelector = (props) => {
+export const FulfillmentSelector = (props: Props) => {
   const {
     productId: productCode,
     fulfillmentMethods,
+    fulfillmentPaths,
     location,
```

Comment:
This component causes a PropType validation warnings `Warning: Failed prop type: The prop location.code is marked as required in FulfillmentSelector, but its value is null.`

It seems that the type definitions don't fit to the default `userLocation` state.

Reply:
done